paimon-web-ui/src/views/playground/components/query/components/console/components/controls/index.module.scss

```diff
@@ -0,0 +1,57 @@
+/* Licensed to the Apache Software Foundation (ASF) under one
+or more contributor license agreements.  See the NOTICE file
+distributed with this work for additional information
+regarding copyright ownership.  The ASF licenses this file
+to you under the Apache License, Version 2.0 (the
+"License"); you may not use this file except in compliance
+with the License.  You may obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing,
+software distributed under the License is distributed on an
+"AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+KIND, either express or implied.  See the License for the
+specific language governing permissions and limitations
+under the License. */
+
+.container {
+  display: flex;
+  align-items: center;
+  width: 100%;
+  height: 42px;
+
+  .left {
+    display: flex;
+    padding: 10px 0 10px 20px;
+  }
+
+  .right {
+    display: flex;
+    flex: 1;
+    justify-content: flex-end;
+    padding: 10px 20px 10px 0;
+  }
+
+  .active-button-light {
+    color: #7D818E;
+    background-color: #DBDCDE;
+    border-radius: 6px;
+    padding: 4px;
+  }
+
+  .active-button-dark {
+    color: #7D818E;
```

Comment:
The same color can be extracted into scss variable.

Reply:
done.